src/components/user/user.ts

```diff
@@ -0,0 +1,41 @@
+import Block from 'core/Block';
+import './users.css';
+import { UsersProps } from './types';
+export class Users extends Block {
+  static componentName = 'Users';
+  constructor({ users, onClick }: UsersProps) {
+    super({ users, events: { click: onClick } });
+  }
+
+  protected getStateFromProps(props: any): void {
+    this.state = {
+      users:
+        props.users !== 'undefined' && props.users.length > 0
```

Comment:
Здесь тоже можно чуточку проще
```suggestion
        props?.users?.length ? ... : ...
```